### DISCUSSION TITLE
Balances - currency options

### DISCUSSION THREAD

**Discussion by mikansontap123** [TomBursch/kitchenowl#141] [Q&A]
Sorry if this is already a thing and I'm just being completely blind - is there any way to change the currency from USD in the "Balances" tool?

**Comment by TomBursch** [comment]
The currency is derived from the app/system language.
For example, if your phone is set to German it will show Euros. There is currently no way to change it other than changing the language. Will put that on my to-do list though.

**Comment by KasperRagnar** [comment]
What is the status of this feature? 

I live in a multilingual home. My parents speak Danish, and my wife and I speak English. 
We all use the same household in the app. 

Would it be possible to make it so that each user selects a language in their user account settings, and the application then translates the currency based on each user's chosen language? 

if not then just allow me to change the language for the whole household
